specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/examples/ApiManagementCreateWorkspacePremiumGatewayKubernetesManaged.json

```diff
@@ -0,0 +1,106 @@
+{
+  "parameters": {
+    "gatewayName": "apimGateway1",
+    "resourceGroupName": "rg1",
+    "api-version": "2024-10-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "parameters": {
+      "properties": {
+        "backend": {
+          "subnet": {
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworks/vn1/subnets/sn1"
+          }
+        },
+        "virtualNetworkType": "External"
+      },
+      "sku": {
+        "name": "WorkspaceGatewayPremium",
+        "capacity": 1
+      },
+      "location": "South Central US",
+      "tags": {
+        "Name": "Contoso",
+        "Test": "User"
+      },
+      "kind": "KubernetesManaged"
+    }
+  },
+  "responses": {
+    "201": {
+      "headers": {
+        "Location": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/gateways/apimGateway1/operationresults/ZWFzdHVzOmFuZHktdGVzdGluZy0yMDIyLTA0LTAxLXByZXZpZXctNF9BY3RfODQ2ZWE4Ng==?api-version=2024-10-01-preview&asyncResponse",
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/gateways/apimGateway1/operationresults/ZWFzdHVzOmFuZHktdGVzdGluZy0yMDIyLTA0LTAxLXByZXZpZXctNF9BY3RfODQ2ZWE4Ng==?api-version=2024-10-01-preview&asyncResponse"
+      },
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/gateways/apimGateway1",
+        "name": "apimGateway1",
+        "type": "Microsoft.ApiManagement/gateway",
+        "tags": {
+          "api-version": "2024-10-01-preview"
+        },
+        "location": "East US",
+        "etag": "AAAAAAAmRAM=",
+        "kind": "KubernetesManaged",
+        "properties": {
+          "provisioningState": "Created",
+          "targetProvisioningState": "Activating",
+          "createdAtUtc": "2022-07-11T18:41:01.2506031Z",
+          "backend": {
+            "subnet": {
+              "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworks/vn1/subnets/sn1"
+            }
+          },
+          "virtualNetworkType": "External"
```

Comment:
```suggestion
```

Reply:
Indeed, looks like I missed the other ones.